src/main.js

```diff
@@ -45,7 +45,8 @@ const faIcons = [
   "Users",
   "VolumeUp",
   "VolumeMute",
-  "VoteYea"
+  "VoteYea",
+  "Transgender"
```

Comment:
Would be awesome if you could keep the list sorted, that makes it easier to find a specific icon.